hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -108,8 +114,10 @@ import qualified Control.Monad.Trans.Writer.Strict as Strict
 
 import qualified Data.Char as Char
 import           Data.Functor.Identity (Identity(..))
+import           Data.Map.Strict (Map)
```

Comment:
Just `Data.Map (Map)` is fine as well. The type between the strict and lazy API is the same.

Reply:
The `Data.Map` suggests using `Data.Map.Strict` when you'll need all values of the map. As such I wanted to be explicit here - but I'd be fine to change it if it's an issue.

Reply:
I'm not suggesting not using the functions from Data.Map.Strict, just noting that the type `Map` is shared, and IMO
```
import           Data.Map (Map)
```
is cleaner